compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -675,7 +677,8 @@ macro_rules! define_queries_struct {
             $(
                 $(#[$attr])*
                 $name: QueryState<
-                    <queries::$name<'tcx> as QueryConfig<QueryCtxt<'tcx>>>::Key
+                    <queries::$name<'tcx> as QueryConfig<QueryCtxt<'tcx>>>::Key,
+                    crate::dep_graph::DepKind,
```

Comment:
Can you use `rustc_middle::dep_graph` instead here? `crate::dep_graph` uses a `use` which isn't very nice.